connect/0.10/Dockerfile

```diff
@@ -1,30 +1,18 @@
 FROM debezium/connect-base:0.10
 
-MAINTAINER Debezium Community
+LABEL maintainer="Debezium Community"
 
-ENV DEBEZIUM_VERSION=0.9.0.Final \
+ENV DEBEZIUM_VERSION="0.9.5.Final" \
```

Comment:
I think that's what you were referring two in your DM, @renatomefi? It will be adjusted once the first 0.10.0 release is out (today or tomorrow hopefully).

Reply:
Yes that's it! Thanks! :)